### PR TITLE
Add documentation for local Keystore signer

### LINKED ISSUE #1993: Document Local Keystore Signer Usage

### PR DESCRIPTION
**Pull request by derekpierre**
Fixes #1993 .

### CODE REVIEW

File: docs/source/guides/ethereum_node.rst [hunk 1]
```diff
@@ -95,39 +95,54 @@ Remote Ethereum Node
...
-external (possibly remote) ethereum node. Clef enables applications to connect to an Ethereum node and send locally signed
-transactions to be broadcasted. This setup could be desirable when interacting with an untrusted Ethereum node. More
-information about Clef can be found `here <https://github.com/ethereum/go-ethereum/tree/master/cmd/clef>`_.
+external (possibly remote) ethereum node and is particularly desirable when interacting with an untrusted
+Ethereum node.
+
+For example, external signers can be used with:
+
+- Infura/Alchemy/Etc.
+- Local geth node
```
Comment (cygnusv) [inline docs/source/guides/ethereum_node.rst:112]: Should we mention parity, or maybe perhaps say "Local ethereum node"?

File: docs/source/guides/ethereum_node.rst [hunk 2]
```diff
@@ -148,19 +163,15 @@ Next, initialize Clef with your chosen password to encrypt the master seed:
...
 .. code:: bash
 
     $ clef --keystore <PATH TO KEYSTORE> --chainid <CHAIN ID> --advanced
 
 
-* <PATH TO KEYSTORE> - The path to the directory containing geth-formatted private key files; the default path for Linux is ``~/.ethereum/keystore``.
-* Chain ID 1 is specified to ensure clef signs transactions with the network ID of mainnet.
-
+* ``<PATH TO KEYSTORE>`` - The path to the directory containing geth-formatted private key files; the default path for Linux is ``~/.ethereum/keystore``.
+* ``<CHAIN ID>`` - 1 is specified to ensure Clef signs transactions with the network ID of mainnet.
```
Comment (cygnusv) [inline docs/source/guides/ethereum_node.rst:174]:
```suggestion
* ``<CHAIN ID>`` - 1 is specified to ensure Clef signs transactions with the network ID of Ethereum mainnet.
```

File: docs/source/guides/ethereum_node.rst [hunk 3]
```diff
@@ -229,3 +240,32 @@ to indicate which are the automated rules (in our case, allowing the listing of
...
+Signing with Local Keystore
+***************************
+
+.. important::
+
+    Can only be used for :ref:`Worker (Ursula) <ursula-config-guide>` operations.
+
+Local keystore signing utilizes `eth-account <https://github.com/ethereum/eth-account>`_ to sign ethereum transactions
+using local ethereum keystore files. By default on Linux, the keystore directory path is ``~/.ethereum/keystore``
+(on MacOS for Goerli testnet, ``/Users/<username>/Library/Ethereum/keystore``).
```
Comment (cygnusv) [inline docs/source/guides/ethereum_node.rst:261]:
```suggestion
(on MacOS, ``/Users/<username>/Library/Ethereum/keystore``).
```

Comment (derekpierre) [reply docs/source/guides/ethereum_node.rst]: See https://github.com/nucypher/nucypher/pull/1945#discussion_r419611966 - that being said, the path is incorrect.

**Comment by KPrasch** [comment]
Merging ( @vepkenez )

**Pull request merged**